sbi/neural_nets/flow.py

```diff
@@ -109,7 +113,7 @@ def build_maf(
                         features=x_numel,
                         hidden_features=hidden_features,
                         context_features=y_numel,
-                        num_blocks=2,
+                        num_blocks=num_blocks,
```

Comment:
So we are changing the default from 2 to 5 blocks?

Reply:
ah, no, default is still 2 blocks because `posterior_nn` will overwrite the default of `build_maf`. Sorry for confusion, will make this more clear in another commit.

Reply:
Note that, for the `made`, the default was 5 blocks, but now it is 2 blocks.

Reply:
Note that, for the made, the default was 5 blocks, but now it is 2 blocks.

Reply:
Let's leave all defaults as they were in this PR. If necessary, we could have another PR that changes defaults